### DISCUSSION TITLE
Cross Probing Sample

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#2007] [Q&A]
in Cross Probing Sample, 
When I click the 3d element, the corresponding two-dimensional element is always selected success!
but When I click the 2D element, the corresponding 3D element is selected, but soon it becomes unselected
 
![12312312312312](https://user-images.githubusercontent.com/18022720/128500314-63fbae63-8e07-4d24-b5e0-17fb2019d3d6.jpg)

**Comment by calebmshafer** [comment]
@Josh-Schifter @StefanRetief
